ross/defects/misalignment.py

```diff
@@ -0,0 +1,1076 @@
+        new_V_dot = (
+            ftmodal
+            + self.Funbmodal[:, i]
+            - ((self.Cmodal + self.Gmodal * self.Omega[i])).dot(velocity)
+            - ((self.Kmodal + self.Kstmodal * self.AccelV[i]).dot(positions))
+        ).dot(self.inv_Mmodal)
+
+        new_X_dot = velocity
+
+        new_Y = np.zeros(24)
+        new_Y[:12] = new_X_dot
+        new_Y[12:] = new_V_dot
+
+        return new_Y
+
+    def _parallel(self, positions, fir):
+        """Reaction forces of parallel misalignment.
+        
+        Returns
+        -------
+        Fmis : array
+            Excitation force caused by the parallel misalignment on the node of application.
+        FFmis : array
+            Excitation force caused by the parallel misalignment on the entire system.
+        """
+        k0 = self.kCOUP
+        delta1 = self.eCOUP
+
+        betam = 0
+
+        k_misalignbeta1 = np.array(
+            [
+                k0 * self.Kcoup_auxI * delta1 * np.sin(betam + fir),
+                -k0 * self.Kcoup_auxI * delta1 * np.cos(betam + fir),
+                0,
+                0,
+                0,
+                0,
+                k0 * self.Kcoup_auxF * delta1 * np.sin(betam + fir),
+                -k0 * self.Kcoup_auxF * delta1 * np.cos(betam + fir),
+                0,
+                0,
+                0,
+                0,
+            ]
+        )
+
+        K_mis_matrix = np.zeros((12, 12))
+        K_mis_matrix[5, :] = k_misalignbeta1
+        K_mis_matrix[11, :] = -k_misalignbeta1
+
+        Force_kkmis = K_mis_matrix.dot(positions[self.DoF])
+
+        F_misalign = np.array(
+            [
+                (-k0 * delta1 * np.cos(betam + fir) + k0 * delta1),
+                -k0 * delta1 * np.sin(betam + fir),
+                0,
+                0,
+                0,
+                self.TD - self.TL,
+                (k0 * delta1 * np.cos(betam + fir) - k0 * delta1),
+                k0 * delta1 * np.sin(betam + fir),
+                0,
+                0,
+                0,
+                -(self.TD - self.TL),
+            ]
+        )
+
+        Fmis = Force_kkmis + F_misalign
+        FFmis = np.zeros(self.ndof)
+        FFmis[self.DoF] = Fmis
+
+        return Fmis, FFmis
+
+
+def base_rotor_example():
+    """Internal routine that create an example of a rotor, to be used in
+    the associated misalignment problems as a prerequisite.
+
+    This function returns an instance of a 6 DoF rotor, with a number of
+    components attached. As this is not the focus of the example here, but
+    only a requisite, see the example in "rotor assembly" for additional
+    information on the rotor object.
+
+    Returns
+    -------
+    rotor : ross.Rotor Object
+        An instance of a flexible 6 DoF rotor object.
+
+    Examples
+    --------
+    >>> rotor = base_rotor_example()
+    >>> rotor.Id[0]
+    0.003844540885417
+    """
+
+    steel = ross.materials.steel
+    steel.rho = 7.85e3
+    steel.E = 2.17e11
```

Comment:
replace `steel` by `steel2`.

Reply:
Hi @rodrigomoliveira1, we've just updated again. Sorry, I did not saw this steel variable in the examples.

Reply:
Hello @rodrigomoliveira1 , we did that locally before lunch break, and noticed it solved a lot of the issues. Some others are coming from unfinished scripts of cracks and rubbing models, as well as from the "draft" script. We are removing them from the folder now and it should clear the problems.

Reply:
> Hi @rodrigomoliveira1, we've just updated again. Sorry, I did not saw this steel variable in the examples.

No problem at all, @marcusfilipesr .

> Hello @rodrigomoliveira1 , we did that locally before lunch break, and noticed it solved a lot of the issues. Some others are coming from unfinished scripts of cracks and rubbing models, as well as from the "draft" script. We are removing them from the folder now and it should clear the problems.

@verg1lio, good to hear it! I'm glad we managed to solve this problem.

Reply:
@verg1lio, I did a bad suggestion in this example, because setting `steel2 = ross.materials.steel` and then chaging `steel2` attributes will also change `ross.materials.steel` attibutes.

To solve this problem, it's necessary to instantiate a new material:
```python
steel2 = ross.Material(name="Steel2", rho=7.85e3, G_s=8.12000e+10, E=2.17e11)
```